src/openmrs-esm-api.ts

```diff
@@ -1,4 +1,20 @@
+import { defineConfigSchema } from "@openmrs/esm-module-config";
 import "./set-public-path";
+
+defineConfigSchema("@openmrs/esm-api", {
+  redirectAuthFailure: {
+    enabled: {
+      default: true
+    },
+    url: {
+      default: "/openmrs/login.htm"
```

Comment:
I would prefer the default to navigate to the esm-login page instead of the refapp login. Additionally, the `/openmrs` prefix is a configurable property within the java server, so we shouldn't hard code it.

We have the [openmrsBase](https://github.com/openmrs/openmrs-module-spa/blob/master/omod/src/main/webapp/master-single-page-application.jsp#L21) property which is that configurable server property. And we have the [getOpenmrsSpaBase](https://github.com/openmrs/openmrs-module-spa/blob/master/omod/src/main/webapp/master-single-page-application.jsp#L23) function available for use, too, when linking within the SPA.

```suggestion
      default: window.getOpenmrsSpaBase() + 'login'
```